aicsimageio/readers/ome_zarr_reader.py

```diff
@@ -135,6 +157,34 @@ def _get_coords(
 
         return coords
 
+    @staticmethod
+    def _get_pixel_size(
+        reader: ZarrReader, dims: List[str], series_index: int, resolution_index: int
+    ) -> Tuple[Optional[float], Optional[float], Optional[float]]:
+
+        coord_transform = reader.root_attrs["multiscales"][series_index]["datasets"][
+            resolution_index
+        ]["coordinateTransformations"]
```

Comment:
According to the spec, there may be one more optional coordinateTransform that needs to be applied if it exists:
```
Each "multiscales" dictionary MAY contain the field "coordinateTransformations", describing transformations that are applied to all resolution levels in the same manner. The transformations MUST follow the same rules about allowed types, order, etc. as in "datasets:coordinateTransformations" and are applied after them. They can for example be used to specify the scale for a dimension that is the same for all resolutions.
```

Reply:
attempted in [6f49b93](https://github.com/AllenCellModeling/aicsimageio/pull/509/commits/6f49b93bc6eec56ab3c76dd910ae48078889a7e5)